Channels/UserInterface/03a_ScheduledUploads.md

```diff
@@ -0,0 +1,98 @@
+[!!Check the offer upload](../Operation/03_CheckOfferUpload.md#check-the-scheduled-offer-uploads)
+
+# Scheduled uploads
+
+*Omni-Channel > Offer upload > Tab SCHEDULED UPLOADS*
+
+![Scheduled uploads](../../Assets/Screenshots/Channels/OfferUpload/ScheduledUploads/ScheduledUploads.png "[Scheduled uploads]")
+
+**Scheduled Exports**
+
```

Comment:
Wir haben es schon kommentiert, aber ich frage nochmals zur Sicherheit. Term ist undeutlich: upload/export, import... Sinnvoll, diese Begriffe zu vereinheitlichen, auch wenn in neuem UI, oder sind sie keine exakten Synonyme?